pkg/server.go

```diff
@@ -118,14 +188,44 @@ func setupRoutes(cfg *Config) (*mux.Router, *PluginConfig) {
 	router := mux.NewRouter()
 
 	router.PathPrefix("/health").HandlerFunc(healthHandler())
+
+	// TODO: needs to check for acm feature and adjust the plugin-manifest to be something appropriate
 	router.Path("/plugin-manifest.json").Handler(manifestHandler(cfg))
```

Comment:
The manifest handler patches the manifest based on the features to add console extensions

Reply:
The use case here will need to be a bit different, as ACM is going to need to trim down and change the features rather than add to them. This functionality falls into [OU-507](https://issues.redhat.com/plugins/servlet/mobile#issue/OU-507) which is the UI part, which I will be starting today. This PR is looking at a separate feature branch, so I don't see this as a blocker for this PR.